setup.py

```diff
@@ -180,6 +180,7 @@
             'jupyter-notebook = notebook.notebookapp:main',
             'jupyter-nbextension = notebook.nbextensions:main',
             'jupyter-serverextension = notebook.serverextensions:main',
+            'jupyter-bundler = notebook.bundlerextensions:main',
```

Comment:
I was staying consistent with `jupyter nbextension` and `jupyter serverextension` which both affect the notebook alone but are top level commands. I don't mind making it a subcommand of notebook but might need a pointer on how that would work. It's not simply renaming the script `jupyter-notebook-bundler`, right?